src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -1834,6 +1835,7 @@ void parser::parse_and_visit_switch(parse_visitor_base &v) {
 
   bool keep_going = true;
   bool is_before_first_switch_case = true;
+  hash_set<string8_view> repeated_cases;
```

Comment:
Nit: I think a better name is `cases` or `case_expressions`. `repeated_cases` sounds like it contains *only* the duplicates, but it actually contains *every* `case`.